docker-compose.yml

```diff
@@ -6,10 +6,13 @@ services:
       dockerfile: Dockerfile
     image: joohoi/acme-dns:latest
     ports:
-      - "443:443"
-      - "53:53"
-      - "53:53/udp"
-      - "80:80"
+      - "443:8443"
+      - "53:5353"
+      - "53:5353/udp"
+      - "80:8080"
     volumes:
       - ./config:/etc/acme-dns:ro
-      - ./data:/var/lib/acme-dns
+      - acmedns_data:/var/lib/acme-dns
```

Comment:
My thought about moving from a data directory to a volume was, that the volume defines automatically the correct permissions that the running container needs to write files into it. In case of the data directory it's the users responsibility to set the correct permissions of the directory.
Hmm, but this change will again be a breaking change like the other ones. It might make sense to revert that change. What do you think?

Reply:
This sounds good. I'm fine with it. If we're breaking backwards compatibility, better to do them all in one occasion. Is there a way to migrate from the previous a-directory-on-host to the new volume based approach, and what would it take?

If there is a migration path, I think we should document it somewhere.

Reply:
Ok. I will document a migration path from the host-directory way to the new volume based approach.

Reply:
Migration is now added into the `README.md` file.